### PR TITLE
added the genepanel: Sujal_Gosh to project Alps

### PR DESCRIPTION
**Pull request by LayalYasin**
Sujal always requests to know the gene coverage for a set of around 350 genes. I added those gene to a list called  Sujal_Gosh.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -127,6 +127,8 @@ projects:
       hhu:
         contact: "Ute Fischer"
         project_id: 2
+      genepanels:
```
Comment (sjanssen2) [inline config.yaml]: the syntax depends on indentation! You need to remove two whitespaces to bring it to the same level as `snupy` and not below `snupy` as it is now

**Comment by sjanssen2** [comment]
I am not totally sure what happens if a gene from the list is NOT contained in the `xxx_Covered.bed` file of the capture kit. Have you checked that all those ~350 names are included?

File: config.yaml (hunk 1 above)
Comment (LayalYasin) [reply config.yaml]: done

Comment (sjanssen2) [reply config.yaml]: :+1:

**Comment by sjanssen2** [comment]
as quick test, could you please generate the file /gpfs/project/projects/spike/Intermediate/092_genepanel_coverage/Sujal_Gosh.yaml/Alps/ALPS_60a.tsv through spike?

**Comment by sjanssen2** [comment]
I tested myself and it seems to work fine :+1 ready to merge. Maybe @deyazoubi will hit the merge button such that he is aware of the changes?!0

**Comment by LayalYasin** [comment]
Hi Stefan,

didnt you face any errors when testing ?

because I have an error:

SyntaxError in line 1099 of /home/yasinl/spike/scripts/reports.py:
invalid syntax (reports.py, line 1099)
  File "/home/yasinl/spike/Snakefile", line 9, in <module>

which refers to the line:
coverage_per_probe.groupby('gene')['percentcovered_0'].mean()], axis=1)
________________________________

**Comment by sjanssen2** [comment]
is it possible that you get these issues because of your changes with the addition of two columns of the bamstat result? It went through for me without any complains. Thus I think this problem is local on your end.

**Comment by sjanssen2** [comment]
p.s. you should always post the full error stack trace, not only the last line to enable better debugging.

**Comment by LayalYasin** [comment]
ah ok, I thought u tested the two additional columns too, thats why I asked


w.r.t the error message, unfortunatey this is the full error stack

SyntaxError in line 1099 of /home/yasinl/spike/scripts/reports.py:
invalid syntax (reports.py, line 1099)
  File "/home/yasinl/spike/Snakefile", line 9, in <module>

I will debug this further
________________________________

**Pull request merged**